### DISCUSSION TITLE
How frequently can / should we meet?

### DISCUSSION THREAD

**Discussion by furkan7258** [ud-turkic/udtw23#37] [Polls]
Dear friends,

How frequently can and should we meet? Is it OK with those of you that attend the weekend meetings that we're meeting on _weekends_? Should we try again to meet during weekdays? Because it's currently on weekends, I know of some members for whom attending is out of question 🙂 .

I'd appreciate your responses to both questions in the comments.

Regards.

**Comment by ctaguchi** [comment]
I voted for weekends but I actually don't have any preference. I'll join it whenever my time zone allows.

**Reply by aslikuzgun** [reply]
+1

**Comment by furkan7258** [comment]
It seems weekdays are preferred more. I've created [a scheduling link](https://www.when2meet.com/?24236201-WEAwl) for next week with only weekdays (Mar 25-29). Hopefully, we'll be able to find a time for our next meeting. I'll also email this link soon.
